packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx

```diff
@@ -0,0 +1,247 @@
+import * as React from 'react';
+import { Popover, PopoverSurface } from '@fluentui/react-popover';
+import { mergeClasses } from '@griffel/react';
+import type { PositioningVirtualElement } from '@fluentui/react-positioning';
+import { tokens } from '@fluentui/react-theme';
+import { useId } from '@fluentui/react-utilities';
+import { getAccessibleDataObject, Points, pointTypes } from '../../utilities/index';
+import { convertToLocaleString } from '../../utilities/locale-util';
+import { Shape } from '../Legends/shape';
+import { usePopoverStyles_unstable } from './usePopoverStyles.styles';
+import { YValueHover } from './CartesianChart.types';
+import { LegendShape } from '../Legends/Legends.types';
+import { PopoverComponentProps } from './Popover.types';
+
+const PopoverComponent: React.FunctionComponent<PopoverComponentProps> = React.forwardRef<
```

Comment:
This is too general purpose a name - Consider ChartPopover along with a summary comment of why you needed to have a Popover different from the Fluent v9 Popover.

Reply:
@srmukher Could you address this comment

Reply:
changed the name to ChartPopover and added comment